inst/tests/tests.Rraw

```diff
@@ -21192,3 +21193,58 @@ test(2319.1, !is.null(attr(dt_get, ".internal.selfref")))
 dt_get0 = data.frame(a = 1:3, b = letters[1:3])
 setDT(get0("dt_get0"))
 test(2319.2, !is.null(attr(dt_get0, ".internal.selfref")))
+
+# 5885 implement frev
+d = c(NA, NaN, Inf, -Inf)
+test(2320.00, frev(c(FALSE, NA)),     c(NA, FALSE))
+test(2320.01, frev(c(0L, NA)),        c(NA, 0L))
+test(2320.02, frev(d),                c(-Inf, Inf, NaN, NA))
+test(2320.03, frev(c(NA, 1, 0+2i)),   c(0+2i, 1, NA))
+test(2320.04, frev(as.raw(0:1)),      as.raw(1:0))
+test(2320.05, frev(NULL),             NULL)
+test(2320.06, frev(character(5)),     character(5))
+test(2320.07, frev(integer(0)),       integer(0))
+test(2320.08, frev(list(1, "a")),     list("a", 1))
+test(2320.09, setrev(c(0L, NA)),      c(NA, 0L))
```

Comment:
Should these be structured more to get after the side-effects, like

```r
test(2320.09, {x = c(0L, NA); setrev(x); x},      c(NA, 0L))
```

More generally, these days I shy away from tests of private API by default (e.g. https://softwareengineering.stackexchange.com/q/380287/221381), should we just drop the `setrev()` tests?

Reply:
done, since we might want to export in the future